lib/ssl/doc/src/ssl.xml

```diff
@@ -1281,7 +1284,7 @@ fun(srp, Username :: binary(), UserState :: term()) ->
 	<desc><p>A server only does x509-path validation in mode
 	<c>verify_peer</c>, as it then sends a certificate request to
 	the client (this message is not sent if the verify option is
-	<c>verify_none</c>).  You can then also want to specify option
+	<c>verify_none</c>, default).  You can then also want to specify option
```

Comment:
```suggestion
	<c>verify_none</c>).  You can then also specify option
```

Reply:
Fixed the option names comment and reworded. Will push new version soon·